File: java/MaratonaJava/javinha/src/main/java/com/maratonajava/estrutura_sequencial/exerc06.java
package com.maratonajava.estrutura_sequencial;

import java.util.Scanner;

public class exerc06 {
    public static void main(String[] args){
        // Faça um Programa que peça o raio de um círculo, calcule e mostre sua área.
        try(Scanner raio = new Scanner(System.in)){
            float valorRaio, calcRaio;

            System.out.println("Programa Calcular o Raio de um círculo.");
            System.out.println("================================================");

            System.out.println("Informe o do Raio de um Círculo em cm.");
            valorRaio = raio.nextFloat();

            //verificar depois mais detalhes desta configuração
            calcRaio =(float) ((valorRaio*valorRaio)*3.14);

            System.out.println("O Valor da área é: "+calcRaio+" cm");
        }
    }
}

File: C/Exercicios/Estrutura_Sequencial/exerc06.c
#include <stdio.h>

#define descricao_programa "Programa que peça o raio de um círculo, calcule e mostre sua área."

int main(){
    printf("%s",descricao_programa);
    //A = π·r2

    float area_circulo, raio_circulo, pi = 3.14159;

    printf("Informe o valor do Raio de um circulo.\n");
    scanf("%f",&raio_circulo);

    area_circulo = pi * (raio_circulo*raio_circulo);

    printf("Valor da Área do Circulo é: %f.\n",area_circulo);

}